docs/reference/ilm/error-handling.asciidoc

```diff
@@ -104,25 +102,21 @@ Which returns the following information:
 --------------------------------------------------
 // TESTRESPONSE[skip:no way to know if we will get this response immediately]
 
-<1> this index is managed by {ilm-init}
-<2> the policy in question, in this case, "shrink-the-index"
-<3> the current age for the index
-<4> what phase the index is currently in
-<5> what action the index is currently on
-<6> what step the index is currently on, in this case, because there is an error, the index is in the "ERROR" step
-<7> the name of the step that failed to execute, in this case "shrink"
-<8> the error class that occurred during this step
-<9> the error message that occurred during the execution failure
-<10> the definition of the phase (in this case, the "warm" phase) that the index is currently on
-
-The index here has been moved to the error step because the shrink definition in
-the policy is using an incorrect number of shards. So rectifying that in the
-policy entails updating the existing policy to use one instead of four for
-the targeted number of shards.
+<1> The policy being used to manage the index: `shrink-index`
+<2> The index age: 5.1 days
+<3> The phase the index is currently in: `warm`
+<4> The current action: `shrink`
+<5> THe step the index is currently in: `ERROR`
+<6> The step that failed to execute: `shrink`
+<7> The type of error that occurred: `illegal_argument_exception`
+<8> The error message that describes what went wrong
+<9> The definition of the current phase from the `shrink-index` policy
+
+To resolve this, you could update the policy to shrink the index to a single shard after 5 days:
```

Comment:
```suggestion
To resolve this issue, update the policy to shrink the index to a single shard after 5 days:
```

Reply:
I'm going to stet this one since it's one possible way to resolve the issue for this particular scenario.